omc3/tbt_converter.py

```diff
@@ -79,10 +79,13 @@ def converter_entrypoint(opt):
 def _read_and_write_files(opt):
     for input_file in opt.files:
         tbt_data = tbt.read_tbt(input_file, datatype=opt.tbt_datatype)
-        for i in range(opt.replications):
-            suffix = f"_{i}" if opt.replications > 1 else ""
-            tbt.write(join(opt.outputdir, f"{_file_name(input_file)}{suffix}"), tbt_data,
-                      noise=opt.noise_to_add)
+        for n in opt.noise_to_add:
```

Comment:
As you allow empty lists with `nargs='*'` I would distinguish here between noise given and not given. The way you have it at the moment will, for example, not add any noise-level into the filename, if you give `noise_to_add = [10.]` even though there is clearly noise

Reply:
That was initially my suggestion, but you are right: if there is noise level specified, i.e. not None, should be in the name.

Reply:
This part can then be a loop over replications, with a check if noise is None inside it...either writes one file or comprehends the writing over list of noise levels...